controllers/machine_controller.go

```diff
@@ -208,14 +208,15 @@ func patchMachine(ctx context.Context, patchHelper *patch.Helper, machine *clust
 	// after provisioning - e.g. when a MHC condition exists - or during the deletion process).
 	conditions.SetSummary(machine,
 		conditions.WithConditions(
-			clusterv1.BootstrapReadyCondition,
```

Comment:
Changing order between InfrastructureReady and BootstrapReady, giving precedence to the first on the given that it usually provides the more relevant feedback